src/hocs/withTokenizeWorker.js

```diff
@@ -53,65 +18,16 @@ export default (worker, options = {}) => {
         return mapPayload(input, props);
     };
 
-    return ComponentIn => class ComponentOut extends Component {
-
-        static displayName = wrapDisplayName(ComponentIn, 'withTokenizeWorker');
+    return ComponentIn => {
+        const ComponentOut = props => {
+            const payload = resolveMessagePayload(props);
```

Comment:
This is the only hoc not moving all logics to hook, things like `mapPayload` (deriving useful objects from props) are always not required in hooks.